skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkikoUIView.kt

```diff
@@ -77,37 +49,43 @@ class SkikoUIView : UIView, UIKeyInputProtocol {
     fun hideScreenKeyboard() = resignFirstResponder()
     fun isScreenKeyboardOpen() = isFirstResponder
 
-    var keyEvent: UIPress? = null
-    private var inputText: String = ""
+    private val pressedKeycodes: MutableSet<Long> = mutableSetOf()
```

Comment:
Instead of storing full event UIPress, store only Long key codes. It fixes memory leak.